.github/workflows/go.yml

```diff
@@ -176,16 +179,6 @@ jobs:
           name: linux-logs
           path: /tmp/logs
 
-      - name: Upload all fuzz corpora
-        if: failure()
-        uses: actions/upload-artifact@v2
-        with:
-          name: fuzz-corpus
-          path: |
-            ${{ env.GOCACHE }}/fuzz
-            **/testdata/fuzz
-            fuzz-corpus
```

Comment:
We don't need that because fuzztool and https://github.com/FerretDB/fuzz-corpus are reliable now